### PR TITLE
Bump version to 0.6.2

### PR DESCRIPTION
**Pull request by gaotongxiao**

### CODE REVIEW

File: README_zh-CN.md [hunk 1]
```diff
@@ -72,9 +72,11 @@ MMOCR 的模块化设计使用户可以定义自己的优化器，数据预处
...
 
+目前我们正同步维护稳定版 (0.6.2) 和预览版 (1.0.0) 的 MMOCR，但稳定版会在 2022 年末开始逐步停止维护。我们建议用户尽早升级至 [MMOCR 1.0](https://github.com/open-mmlab/mmocr/tree/1.x)，以享受到由新架构带来的更多新特性和更佳的性能表现。阅读我们的[维护计划](https://mmocr.readthedocs.io/zh_CN/dev-1.x/migration/overview.html)以了解更多信息。
+
 ### 💎 稳定版本
 
-最新的月度版本 v0.6.1 在 2022.08.04 发布。
+最新的月度版本 v0.6.2 在 2022.10.13 发布。
 
 1. 对文本检测和识别任务，新支持了 ArT 数据集。
 2. 修复了多个可能影响模型正确性的错误。
```
Comment (xinke-wang) [inline README_zh-CN.md:82]: new features in Chinese

File: README.md [hunk 2]
```diff
@@ -71,19 +71,20 @@ The main branch works with **PyTorch 1.6+**.
...
-3. Thanks to [MIM](https://github.com/open-mmlab/mim), our installation is much simpler now! The [docs](https://mmocr.readthedocs.io/en/latest/install.html) has been renewed as well.
+1. It's now possible to train/test models through Python Interface.
+2. ResizeOCR now fully supports all the parameters in mmcv.impad.
 
 Read [Changelog](https://mmocr.readthedocs.io/en/latest/changelog.html) for more details!
 
 ### 🌟 Preview of 1.x version
 
-A brand new version of **MMOCR v1.0.0rc0** was released in 2022-09-01:
+A brand new version of **MMOCR v1.0.0rc1** was released in 2022-10-09:
```
Comment (xinke-wang) [inline README.md:87]: also change Chinese version to rc1?

File: docs/en/changelog.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
 # Changelog
 
+## 0.6.2 (13/08/2022)
+
+### Highlights
+
+It's now possible to train/test models through Python Interface. For example, you can train a model under mmocr/ directory in this way:
+
+```python
+an example of how to use such modifications is shown as the following:
```
Comment (xinke-wang) [inline docs/en/changelog.md:10]:
```suggestion
# an example of how to use such modifications is shown as the following:
```

File: docs/en/changelog.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+
+- Adding test & train API to be used directly in code by @wybryan in https://github.com/open-mmlab/mmocr/pull/1138
+- Let ResizeOCR full support mmcv.impad's pad_val parameters by @hsiehpinghan in https://github.com/open-mmlab/mmocr/pull/1437
+
+### Bug Fixes
+
+- Fix ABINet config by @gaotongxiao in https://github.com/open-mmlab/mmocr/pull/1256
+- Fix Recognition Score Normalization Issue by @xinke-wang in https://github.com/open-mmlab/mmocr/pull/1333
+- Remove max_seq_len inconsistency by @antoniolanza1996 in https://github.com/open-mmlab/mmocr/pull/1433
+- box points ordering  by @yjmm10 in https://github.com/open-mmlab/mmocr/pull/1205
```
Comment (xinke-wang) [inline docs/en/changelog.md:32]:
```suggestion
- box points ordering by @yjmm10 in https://github.com/open-mmlab/mmocr/pull/1205
```

**Review by xinke-wang — APPROVED** [review]

**Review by Harold-lkk — APPROVED** [review]

**Review by Harold-lkk — APPROVED** [review]

**Pull request merged**